test/utils/getDefaultCommitContent.spec.js

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs'
+import getDefaultCommitContent from '../../src/utils/getDefaultCommitContent'
+
+describe('getDefaultCommitContent', () => {
+  it('should retrieve title and message when file exists', () => {
```

Comment:
I would move those it on different describes:

describe(when file exists)
  it(should retreive title and message)

describe(when file does not exists)
  it(should retrieve ...)